samples/helloworld/README.md

```diff
@@ -576,6 +576,16 @@ Installed: /home/yourname/openenclave/share/openenclave/samples/lvi_mitigation_b
 The directory `/home/yourname/openenclave/share/openenclave/samples/lvi_mitigation_bin` should contain all
 the dependencies.
 
+#### CMake
+
+```bash
+mkdir build
+cd build
+cmake -DLVI_MITIGATION=ControlFlow -DLVI_MITIGATION_BINDIR=/home/yourname/openenclave/share/openenclave/samples/lvi_mitigation_bin ..
```

Comment:
Consider use \ to break lines, for example:
cmake \
-DLVI_MITIGATION=ControlFlow \
-DLVI_MITIGATION_BINDIR=/home/yourname/openenclave/share/openenclave/samples/lvi_mitigation_bin